### PR TITLE
[add] Multiple focus support

### PR DESCRIPTION
**Pull request by nhunzaker**
focus

### CODE REVIEW

File: example/index.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+var React  = require('react');
+var FocusTrap = require('../src/FocusTrap');
+
+var Component = React.createClass({
```
Comment (averyvery) [inline example/index.jsx:4]: :+1: for example.

File: example/index.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+var React  = require('react');
+var FocusTrap = require('../src/FocusTrap');
```
Comment (averyvery) [inline example/index.jsx]: lose the semis?

**Comment by averyvery** [comment]
:+1:

File: example/index.jsx (hunk 2 above)
Comment (nhunzaker) [inline example/index.jsx]: You don't have to ask me twice!

Comment (nhunzaker) [inline example/index.jsx]: Done

**Pull request closed without merging**